anilist/queries/__init__.py

```diff
@@ -2,9 +2,41 @@
 # SPDX-License-Identifier: MIT
 # Copyright (C) 2021 Amano Team <https://amanoteam.com/> and the python-anilist contributors
 
-from importlib.resources import read_text
 from ._query_files import activity, get, search
 
+try: # because we're supposed to support Python 3.6. I don't get it, but ok.
+    from importlib.resources import read_text
+except ImportError: # re-implement some importlib stuff
+    from typing import TYPE_CHECKING, Union
+
+    if TYPE_CHECKING:
```

Comment:
What if `TYPE_CHECKING` is `False`? `Package` and `Resource` were left undefined.